docs-new/docs/graphql/core/api-reference/graphql-api/query.mdx

```diff
@@ -593,11 +561,7 @@ operators](https://www.postgresql.org/docs/current/functions-comparison.html)
 and [list based search
 operators](https://www.postgresql.org/docs/current/functions-comparisons.html).)
 
-<div id="text_operators">
-
-**Text related operators:**
-
-</div>
+**Text related operators:** {#text-operators}
```

Comment:
same here and for rest of file

try empty div

Reply:
Done